### ISSUE TITLE
Scrollable region must have keyboard access

### ISSUE THREAD

**Issue by evanSe** [handsontable/handsontable#10544]
**Describe the bug**
This is a violation of 2.1.1 Keyboard

**potential solution**
could we add a tabindex="0" to `<div class="wtHolder" role="presentation" style="position: relative; height: 464px; width: 1481px;">`

**Report source**
Internal

**Handsontable version**
    "handsontable": "0.0.0-next-9379dd1-20231020"

environment: 
**To Reproduce**
1. clone code from https://stackblitz.com/github/handsontable/handsontable/tree/feature/accessibility/examples/next/docs/js/accessibility?file=README.md
2.  Audit accessibility of both lighthouse and axe dev tools

you should have the following errors 
-  [Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMkZaQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1e30b8e17581bb976bf16f4b8267eb87294b6338/image.png)
-  [Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMk5aQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1ad702dc4184fedc1ea5ac160829107ce2c5920c/image.png)

**Expected behavior**
do not violate 
- 2.1.1 Keyboard
